node/src/evm_tracing_types.rs

```diff
@@ -0,0 +1,108 @@
+// Copyright 2020-2023 Litentry Technologies GmbH.
```

Comment:
We should use new entity name here https://github.com/litentry/litentry-parachain/issues/2086

Reply:
I see. I will update new files accordingly.